components/ws-manager-api/go/crd/v1/workspace_types.go

```diff
@@ -168,6 +168,17 @@ func (ps PortSpec) Equal(other PortSpec) bool {
 	return true
 }
 
+type WorkspaceImageInfo struct {
+	// +kubebuilder:validation:Required
+	TotalSize int64 `json:"totalSize"`
+
+	// +kubebuilder:validation:Optional
+	WorkspaceImageSize int64 `json:"workspaceImageSize,omitempty"`
```

Comment:
I wonder whether/how we could make it more clear what these fields mean. Technically the names make sense, but if we show this to users, they need an explanation.

But maybe that's fine, and we can write a short paragraph in the docs. And think about additional ways to display it. :+1: